### ISSUE TITLE
Compilation failure

### ISSUE THREAD

**Issue by bhecquet** [SeleniumHQ/htmlunit-driver#116]
Hello

from today and the release of selenium-java 4.1.3, htmlunit driver fails with the following error:
```
java.lang.NoSuchMethodError: org.openqa.selenium.remote.DesiredCapabilities.htmlUnit()Lorg/openqa/selenium/remote/DesiredCapabilities;
```

This is due to this commit https://github.com/SeleniumHQ/selenium/commit/a14ed614a9d15a48763469288fada64257d9fb6a which removed the "htmlunit()" method from DesiredCapabilities

One solution would be to use in HtmlUnitDriver.java:542

```
DesiredCapabilities capabilities = new DesiredCapabilities(HTMLUNIT.browserName(), "", Platform.ANY)
```

instead of 

```
DesiredCapabilities capabilities = DesiredCapabilities.htmlUnit();
```

I've not checked if it's used in an other place

Regards

**Comment by rbri** [comment]
Will address this for the next release....

**Comment by rbri** [comment]
Should be fixed with 3.61.0
